cmd/api/main.go

```diff
@@ -115,15 +116,35 @@ func main() {
 		log.Fatal().Err(err).Msg("parsing http rate lim interval")
 	}
 
-	router := router.ConfiguredRouter(
+	instrUserStore, err := sqlstoreimpl.NewInstrumentedUserStore(userStore)
+	if err != nil {
+		log.Fatal().Err(err).Msg("creating instrumented user store")
+	}
+
+	mesaService := impl.NewTablelandMesa(parser, instrUserStore, chainStacks)
+	mesaService, err = impl.NewInstrumentedTablelandMesa(mesaService)
+	if err != nil {
+		log.Fatal().Err(err).Msg("instrumenting mesa")
+	}
+
+	stores := make(map[tableland.ChainID]sqlstore.SystemStore, len(chainStacks))
+	for chainID, stack := range chainStacks {
+		stores[chainID] = stack.Store
+	}
+	systemService, err := systemimpl.NewSystemSQLStoreService(
+		stores,
 		config.Gateway.ExternalURIPrefix,
 		config.Gateway.MetadataRendererURI,
-		config.HTTP.MaxRequestPerInterval,
-		rateLimInterval,
-		parser,
-		userStore,
-		chainStacks,
 	)
+	if err != nil {
+		log.Fatal().Err(err).Msg("creating system store")
+	}
+	systemService, err = systemimpl.NewInstrumentedSystemSQLStoreService(systemService)
+	if err != nil {
+		log.Fatal().Err(err).Msg("instrumenting system sql store")
+	}
+
+	router := router.ConfiguredRouter(mesaService, systemService, config.HTTP.MaxRequestPerInterval, rateLimInterval)
```

Comment:
We now need to build these dependencies at a higher level to pass into `router.ConfiguredRouter` because of the API change there.